pycbc/waveform/waveform.py

```diff
@@ -927,6 +927,37 @@ def get_waveform_filter_length_in_time(approximant, template=None, **kwargs):
     else:
         return None
 
+def get_waveform_from_ascii(waveform, deltaT, epoch=0.0):
```

Comment:
The "waveform" argument doesn't seem to be used. Why is it needed? Also, is there no way to get delta t from the txt file? Is there a way to get this information from whatever is in the first column?

Reply:
The waveform argument supposedly contains the path to the text file (but lalapps_inspinj doesnt allow me to do that) 
Yes we can simply calculate the delta T from the time column its simple